Resources/config/oauth.xml

```diff
@@ -95,7 +95,12 @@
         -->
         <service id="hwi_oauth.authentication.listener.oauth" class="%hwi_oauth.authentication.listener.oauth.class%"
                  parent="security.authentication.listener.abstract" public="false" abstract="true" />
-        <service id="hwi_oauth.authentication.provider.oauth" class="%hwi_oauth.authentication.provider.oauth.class%" public="false" />
+        <service id="hwi_oauth.authentication.provider.oauth" class="%hwi_oauth.authentication.provider.oauth.class%" public="false">
```

Comment:
@ilirhxh This is managed by internal factory, so it should be reverted & adjusted in:
```
HWI\Bundle\OAuthBundle\DependencyInjection\Security\Factory\OAuthFactory::createAuthProvider()
# line 107-108
->addArgument(new Reference('security.token_storage'))